src/basic/basic.fsproj

```diff
@@ -93,6 +93,8 @@
     <Compile Include="FStar.Options.fs" />
     <Compile Include="FStar.Unionfind.fsi" />
     <Compile Include="FStar.Unionfind.fs" />
+    <Compile Include="boot/FStar.BigInt.fsi" />
+    <Compile Include="boot/FStar.BigInt.fs" />
```

Comment:
FYI, all the files in the `boot` directory need to be adjacent to each other in the .fsproj file. Otherwise, visual studio will refuse to open it.